Automattic/vip-go-mu-plugins
# Search: Do not alert if index is in midst of being re-built

## Description
We don't need multiple alerts firing on index inconsistencies if the active index is already being re-built.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1. Fake index shard requirements not being met:
```
add_filter( 'ep_post_mapping', function( $mapping ) {
    $mapping['settings']['index.number_of_shards'] = 3;
return $mapping;
}, 20 );
```
2. Fake index in the process of being built:
```
update_option( 'vip_search_cron_new_version_building_lock', true );
```
3. Throw a die in the condition
4. Shell in and kick off a health check: 
```
$es = new \Automattic\VIP\Search\Search();
$es->init();
$hj = new \Automattic\VIP\Search\SettingsHealthJob( $es );
$hj->check_settings_health();
```
